src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,55 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
```

Comment:
Maybe you could rephrase this as "Changes and additions are being made to compliance with accesibility standards." because it is a change for the Preferences Framework in general.